DOC.md

```diff
@@ -1238,10 +1245,15 @@ any way, correspond to the jump-index of the nodes!
   	when passing multiline strings via `[[]]` (default true).
   * `dedent`: remove indent common to all lines in `format`. Again, makes
   	passing multiline-strings a bit nicer (default true).
+  * `repeat_duplicates`: repeat nodes when a key is reused instead of copying
```

Comment:
Oh, best add a "when possible", for example functionNode can, and will, not be repeated

Reply:
ok, i’ll add it when i’m back from work